anyway/__init__.py

```diff
@@ -1 +1,5 @@
-from anyway.flask_app import app
+try:
+    from anyway.flask_app import app
+except ModuleNotFoundError:
+    pass
+
```

Comment:
What are the locations that use this import?

Reply:
I didn't check this as the goal in this PR is to move forward with anyway-etl with minimal changes

I agree it's worth to check all the imports and improve them to reduce dependencies

I opened a new issue for this - #1876